src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -19,6 +20,8 @@
  *      specifically for rank questions.
  */
 public class FeedbackRankQuestionUiTest extends FeedbackQuestionUiTest {
+    private static final int NEW_QUESTION_INDEX = -1;
```

Comment:
The design for new question index isn't very ideal but I think a fair change would be to simply make `InstructorFeedbackEditPage#NEW_QUESTION_NUM` public.